app/components/progress_bar_component.rb

```diff
@@ -0,0 +1,26 @@
+class ProgressBarComponent < ViewComponent::Base
+    PROG_TITLE_TO_GROUPINGS = {
+      "Teach primary computing" => [
+        "Contribute to an online discussion",
+        "Develop your teaching practice",
+        "Develop computing in your community"
+      ],
+      "Teach secondary computing" => [
+        "Develop your subject knowledge",
+        "Develop your teaching practice",
+        "Develop computing in your community"
+      ]
+    }.freeze
```

Comment:
These groupings already exist in the database, so it would be great to pull these from the dataase, such as by the code I suggest below

2 reasons why this would be good: it DRYs up the copy so that it only needs to be changed in one place, reducing effort and the chance of mistakes, and secondly, when we use the component on another page, such as the A Level and IB pages that we are building very soon, you won't need to hardcode several lines of new data into this class.

Reply:
resolved thank you